src/index.js

```diff
@@ -0,0 +1,118 @@
+import stringify from "json-stable-stringify";
+
+/**
+ * Properties described in the NPM package.json specification.
+ * @see https://docs.npmjs.com/cli/v9/configuring-npm/package-json
+ * @type {string[]}
+ */
+const packageJsonStandardProperties = [
+  "name",
+  "version",
+  "description",
+  "keywords",
+  "homepage",
+  "bugs",
+  "license",
+  "author",
+  "contributors",
+  "funding",
+  "files",
+  // "exports", // Sort along with `imports` in the Node.js section.
+  "main",
+  "browser",
+  "bin",
+  "man",
+  "directories",
+  "repository",
+  "scripts",
+  "config",
+  "dependencies",
+  "devDependencies",
+  "peerDependencies",
+  "peerDependenciesMeta",
+  "bundleDependencies",
+  "optionalDependencies",
+  "overrides",
+  "engines",
+  "os",
+  "cpu",
+  "devEngines",
+  "private",
+  "publishConfig",
+  "workspaces",
+];
+/**
+ * Properties used by Node.js.
+ * @see https://nodejs.org/api/packages.html#nodejs-packagejson-field-definitions
+ * @type {string[]}
+ */
+const packageJsonNodeJsProperties = [
+  "packageManager",
+  "type",
+  "exports",
+  "imports",
+];
+
+const rootProperties = /** @type {string[]} */ ([]).concat(
+  packageJsonStandardProperties,
+  packageJsonNodeJsProperties,
+);
+
+/**
+ * Returns a new object that has the same properties as the given object but
+ * sorted according the properties list. Any unlisted properties are just added
+ * to the end.
+ *
+ * @param {object} object
+ * @param {string[]} properties
+ * @returns {object}
+ */
+function sortProperties(object, properties) {
+  const sortedObject = {};
+  // Add the specified properties in the given order then all the others in the
+  // order they appear
+  properties.concat(Object.keys(object)).forEach(function (property) {
+    if (!Object.hasOwn(object, property)) {
+      return;
+    }
+    const value = object[property];
+    if (
+      Array.isArray(value) &&
+      value.every((element) => typeof element === "string")
+    ) {
+      // Sort the arrays of strings like `keywords`, `files`
+      sortedObject[property] = value.sort();
+      return;
+    }
```

Comment:
Would this sort strings in arrays that are not standard `package.json` properties? 
For example, an array could be an argument list for some config for a package (Think like `prettier` allows for custom config in `package.json`) , and we shouldn't update the array order. 

Is that a possible scenario?

Reply:
It will sort all arrays of strings in the root of the `package.json`. Arrays of objects or not in the root will not be modified. So yes, it will sort an array of strings of a non-standard property in the root. Is there any specific case for this?

The config options of Prettier is an object. Husky and lint-staged also use objects with keys that may have arrays. Same as ESLint. None of those will be altered.

Reply:
ah, I thought it would run on nested properties. That's why I was checking. AFAIK all config options tend to use an object, but you never know 🤷🏽